test/csuite/schema_abort/main.c

```diff
@@ -612,41 +620,50 @@ thread_ckpt_run(void *arg)
             }
         }
 
-        /*
-         * Set the configurations. Set use_timestamps regardless of whether timestamps are in use.
-         * Set flush_tier even if tiered is not set in the program as it should be a no-op in that
-         * case. Only report that flush is in use in the program output if tiered is actually being
-         * used however.
+        /* Set the configurations. Set use_timestamps regardless of whether timestamps are in use.
          */
         testutil_check(session->checkpoint(session, flush_tier ? ckpt_flush_config : ckpt_config));
 
+        /*
+         * We're ready to be killed after the first checkpoint, or if tiered storage, after the
+         * first flush_tier has been initiated. However, if we have a stop timestamp, we are ready
+         * if the stable has reached the requested stop timestamp.
```

Comment:
Minor suggestion: Since `stop_timstamp` is the most significant decision here I'd rework the comment and the code to put it first. Then go on about the other conditions like checkpoint or flush tier.

Reply:
fixed.